herbert_core/classes/MPIFramework/@MessagesFilebased/private/receive_message_.m

```diff
@@ -131,10 +138,12 @@
 err_code  =MESS_CODES.ok;
 err_mess=[];
 
-if is_failed  % make failed message persistent
-    plo = unlock_(rlock_file);
-    return;
-end
+obj.check_set_persistent(message,mid_from(1));
+
+% if is_failed  % make failed message persistent
```

Comment:
commented code block

Reply:
leftover from the previous implementation of persistent messages, left as the reminder on how the things were done earlier. Probably can be deleted now